### PR TITLE
add prisma tree-sitter and lsp support

### PR DESCRIPTION
**Pull request by seshotake**

### CODE REVIEW

File: languages.toml [hunk 1]
```diff
@@ -1448,3 +1448,17 @@ indent = { tab-width = 2, unit = "\t" }
 [[grammar]]
 name = "openscad"
 source = { git = "https://github.com/bollian/tree-sitter-openscad", rev = "5c3ce93df0ac1da7197cf6ae125aade26d6b8972" }
+
+[[language]]
+name = "prisma"
+scope = "source.prisma"
+injection-regex = "prisma"
+file-types = ["prisma"]
+roots = [".git", "package.json"]
```
Comment (the-mikedavis) [inline languages.toml:1457]:
```suggestion
roots = ["package.json"]
```

we check for `.git` as a default for all languages so no need to add it here: https://github.com/helix-editor/helix/blob/4a27e2d93846f3b66ab0dc225e65d2fb869463f3/helix-loader/src/lib.rs#L87

**Comment by the-mikedavis** [comment]
Overall this looks good but we may want to await https://github.com/victorhqc/tree-sitter-prisma/issues/34. I can reproduce https://github.com/victorhqc/tree-sitter-prisma/issues/34#issuecomment-1148251194 within helix.

File: languages.toml (hunk 1 above)
Comment (seshotake) [reply languages.toml]: fixed

File: languages.toml [hunk 2]
```diff
@@ -1448,3 +1448,17 @@ indent = { tab-width = 2, unit = "\t" }
...
+injection-regex = "prisma"
+file-types = ["prisma"]
+roots = ["package.json"]
+comment-token = "//"
+language-server = { command = "prisma-language-server", args = ["--stdio"] }
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "prisma"
+source = { git = "https://github.com/victorhqc/tree-sitter-prisma", rev = "0ac307e4be0409d4327082ddc737ed432a49dfc3" }
```
Comment (the-mikedavis) [inline languages.toml:1464]:
```suggestion
source = { git = "https://github.com/victorhqc/tree-sitter-prisma", rev = "a3eaf13e75683cb22269814bf5e87af7eb671b1c" }
```

That issue got fixed so with this I think we'll be good to go 🙂

Comment (seshotake) [reply languages.toml:1464]: I'm not sure that got fixed

Comment (victorhqc) [reply languages.toml:1464]: Can you check again with this commit? `36a8e66164babe9f588489c4d9f2b11ddb520ff3` If it stops happening, then I'll merge [this PR](https://github.com/victorhqc/tree-sitter-prisma/pull/39) and build a new release.

Comment (victorhqc) [reply languages.toml:1464]: Released [v1.3.0](https://github.com/victorhqc/tree-sitter-prisma/releases/tag/v1.3.0) should not have the seg fault anymore.

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks for working on this, looks great!

**Comment by seshotake** [comment]
I added to wiki how to install default language server.

**Pull request merged**